### ISSUE TITLE
v2 broken on ubuntu-latest

### ISSUE THREAD

**Issue by miluoshi** [nrwl/nx-set-shas#10]
After last update my calls to `nrwl/nx-set-shas@v2` fail with following error:
```
Error: Not Found
Error: Process completed with exit code 1.
```

After reverting to the previous version `@v2.0.2` it works as before.

I'm calling the action with following input:
```
- uses: nrwl/nx-set-shas@v2
  with:
    main-branch-name: master
```

**Comment by attilacsanyi** [comment]
I have the same issue @miluoshi , I hope they will fix it soon, meanwhile with `nrwl/nx-set-shas@v2.0.2 ` is working
**Saved my day big thanks!** _(I was not aware I can set this fine grained version, not just v2)_

**Comment by miluoshi** [comment]
@attilacsanyi you can use any git ref as a version - tag, branch name or even specific commit hash. You can find list of available tags here:
![IMG_20211217_143546.jpg](https://user-images.githubusercontent.com/1130547/146552685-3a86e206-1b6d-45cc-a94a-9c25ff45f421.jpg)

**Comment by meeroslav** [comment]
Sorry folks for that.
The wrong order of params had escaped me today.

Thanks, @miluoshi for reporting the issue.

The latest version v2.1.2 should fix the issue with (#11)
